pkg/util/bijection/bijection.go

```diff
@@ -107,3 +109,102 @@ func (m *Bijection[T1, T2]) DeleteT2(key T2) bool {
 
 	return true
 }
+
+func NewOneToManyRelation[T1, T2 comparable]() *OneToManyRelation[T1, T2] {
```

Comment:
This is not a bijection. Pls put it into a separate package.

Reply:
The code has been updated.  I temporarily name the multi-level mapping as "MultiMap." Do you have any better ideas.